apps/website/src/pages/docs/_layout.astro

```diff
@@ -248,6 +248,10 @@ const currentPage = new URL(Astro.request.url).pathname;
           display: grid;
           gap: var(--space-1);
         }
+
+        :global(.expressive-code) {
+          overflow: auto;
+        }
```

Comment:
Is this needed? I don't think I noticed any difference on the ThemeProvider docs page after commenting this out.

So I guess this is something pre-emptive, that will be needed for the future?

Reply:
try it on pages that have longer lines of code. resize your browser to be narrower if that helps